packages/cli/src/tools/packageManager.js

```diff
@@ -11,7 +12,12 @@ type Options = {|
 
 let projectDir;
 
-function executeCommand(
+export const ERRORS = {
```

Comment:
@grabbou: not entirely sure if this is suitable for the current CLI code, open for opinions.

Reply:
We've been subclassing errors so far - I guess you could try that and just go with `instanceof` check.